frontend/src/app/main/ui/dashboard/comments.cljs

```diff
@@ -63,7 +64,18 @@
         (mf/use-callback
          (fn [thread]
            (st/emit! (-> (dwcm/navigate thread)
-                         (with-meta {::ev/origin "dashboard"})))))]
+                         (with-meta {::ev/origin "dashboard"})))))
+
+        on-read-all
+        (mf/use-callback
+         (mf/deps team-id)
+         (fn []
+           (st/emit! (dcm/mark-all-threads-as-read
+                      team-id
+                      #(st/emit! (ntf/show {:level :info
```

Comment:
This can be directly handled on the `mark-all-threads-as-read` event and avoid this callback